crates/ruff_linter/src/rules/airflow/rules/dag_schedule_argument.rs

```diff
@@ -0,0 +1,113 @@
+use ruff_diagnostics::{Diagnostic, Violation};
+use ruff_macros::{derive_message_formats, violation};
+use ruff_python_ast::Expr;
+use ruff_python_ast::{self as ast, Keyword};
+use ruff_text_size::Ranged;
+
+use crate::checkers::ast::Checker;
+
+/// ## What it does
+/// Checks that the `DAG()` class or a `@dag()` decorator has an explicit
+/// `schedule` parameter.
```

Comment:
```suggestion
/// Checks that the `DAG()` class or a `@dag()` decorator without an explicit
/// `schedule` parameter.
```

It then reads better with *Why is this bad*